cds/config.py

```diff
@@ -929,7 +1105,7 @@ def _parse_env_bool(var_name, default=None):
     "base-uri": ["'self'"],
     "worker-src": ["'self'", "blob:"],
     "manifest-src": ["'none'"],
-    "prefetch-src": ["'none'"],
+    # "prefetch-src": ["'none'"],
```

Comment:
on  purpose?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/prefetch-src , it is deprecated. I will remove the comment.